src/measure_theory/bochner_integration.lean

```diff
@@ -428,24 +428,18 @@ subtype.ext_iff.symm
 @[norm_cast] protected lemma eq_iff' {f g : α →₁ₛ[μ] E} : (f : α →ₘ[μ] E) = g ↔ f = g :=
 iff.intro (simple_func.eq') (congr_arg _)
 
-/-- L1 simple functions forms a `emetric_space`, with the emetric being inherited from L1 space,
-  i.e., `edist f g = ∫⁻ a, edist (f a) (g a)`.
-  Not declared as an instance as `α →₁ₛ[μ] β` will only be useful in the construction of the Bochner
-  integral. -/
-protected def emetric_space  : emetric_space (α →₁ₛ[μ] E) := subtype.emetric_space
+local attribute [instance, priority 10000]
```

Comment:
What does this do, with no name after it?

Reply:
Oops, leftover from the refactor, probably. I have removed it. Thanks for spotting this!